faq.md

```diff
@@ -78,6 +81,32 @@ and more. Use cases include complex monitoring and analytics; predicting the per
 behavior of applications, models, consumers and connected machines; powering operational
 analytical workflows and dashboards; for QA and performance testing. [[Top]](#top)
 
+### **Why should I choose Timescale Cloud?** [](why-cloud)
+If you want all the benefits of TimescaleDB without the hassle of installing,
+maintaining, and administering the database itself, then Timescale Cloud is the
+best choice for you.
+
+Timescale Cloud manages all of the operational elements of your database so you can 
+focus on building your applications and not making sure the infrastructure works. We 
+ensure you have a secure, high availability environment where we manage the infrastructure 
+all the way down to setting up replications, point-in-time recovery, read replicas, 
+backups, and more.
```

Comment:
Please add 'Top' link to the end of all new FAQ entries (see other entries for example)